integration-tests/src/test/java/uk/gov/di/authentication/api/AuthorisationIntegrationTest.java

```diff
@@ -249,11 +250,15 @@ void shouldRedirectToLoginWithSamePersistentCookieValueInRequest() {
                 redirectUri,
                 startsWith(TEST_CONFIGURATION_SERVICE.getAuthFrontendBaseURL().toString()));
         assertThat(
-                response.getMultiValueHeaders().get(ResponseHeaders.SET_COOKIE).size(), equalTo(2));
+                response.getMultiValueHeaders().get(ResponseHeaders.SET_COOKIE).size(), equalTo(3));
         assertThat(
                 getHttpCookieFromMultiValueResponseHeaders(response.getMultiValueHeaders(), "gs")
                         .isPresent(),
                 equalTo(true));
+        assertThat(
+                getHttpCookieFromMultiValueResponseHeaders(response.getMultiValueHeaders(), "bsid")
+                        .isPresent(),
+                equalTo(true));
```

Comment:
Picky but this could just be 
```
assertTrue(getHttpCookieFromMultiValueResponseHeaders(response.getMultiValueHeaders(), "bsid").isPresent())

```